web_widget_image_download/static/src/js/web_widget_image_download.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+ */
+odoo.define('web_widget_image_download', function (require) {
```

Comment:
Module name should follow [the convention `module_name.servie`.](https://www.odoo.com/documentation/9.0/reference/javascript.html#javascript-module-system-overview)